ldm/modules/encoders/modules.py

```diff
@@ -245,10 +244,14 @@ def __init__(
     ):
         super().__init__()
         self.tokenizer = CLIPTokenizer.from_pretrained(
-            version, local_files_only=True
+            version,
+            cache_dir=os.path.join(Globals.root,'models',version),
```

Comment:
Introduce variable for the `os.path.join` and then use that in these two calls.

Reply:
+1

Reply:
Done.